src/Core/Helpers/Helpers.cs

```diff
@@ -67,5 +72,35 @@ internal static string TrimTrailingSlashes(this string s)
 
             return s.Substring(0, s.Length - count);
         }
+
+        internal static HttpContent ToHttpContent(this IFormCollection collection, string contentType)
+        {
```

Comment:
Can you add just a few comments?

Reply:
Done, I hope they're what you're after. I've also made a few other changes, including at least one that was a fix.

Unfortunately that for some reason has created a code coverage issue for the invalid content type path, and said path is impossible to hit due to how this is used...

Reply:
That's fine.  I can write unit test to wriggle in there.